src/enlyze/models.py

```diff
@@ -300,13 +302,15 @@ def to_dataframe(self) -> pandas.DataFrame:
         ``end`` of every production run will be represented as :ref:`timezone-aware
         <python:datetime-naive-aware>` :py:class:`datetime.datetime` localized in UTC.
 
-        :returns: DataFrame with production runs
-
+        :returns: DataFrame with production runs.
         """
         if not self:
             return pandas.DataFrame()
 
-        df = pandas.json_normalize([asdict(run) for run in self])
+        path_separator = "."
```

Comment:
Was there a reason why we chose "." as a separator? I made the experience lately that some databases don't allow "." in column names and convert them to underscores. We don't need to change that, I'm just curious about our reasoning.

Reply:
The `.` is the default by pandas which we didn't change when introducing `to_dataframe()`. I'm keeping it and make it explicit in order not to introduce a breaking change for the users. That being said, I can imagine that `_` would also feel more natural than the `.` . I'm open to changing it, or maybe we make it configurable in `to_dataframe()` as well?